inference-engine/src/mkldnn_plugin/nodes/math.cpp

```diff
@@ -88,7 +85,23 @@ class MathImpl: public ExtLayerBase {
             else
                 THROW_IE_EXCEPTION << layer->name << " Incorrect Math layer type!";
 
-            addConfig(layer, { { ConfLayout::PLN, false, 0 } }, { { ConfLayout::PLN, false, 0 } });
+            LayerConfig config;
+            DataConfig inConfig;
+            inConfig.inPlace = 0;
+            inConfig.constant = false;
+            const SizeVector& inDims = layer->insData[0].lock()->getTensorDesc().getDims();
+            inConfig.desc = TensorDesc(Precision::FP32, inDims, InferenceEngine::TensorDesc::getLayoutByDims(inDims));
+            config.inConfs.push_back(inConfig);
+
+            DataConfig outConfig;
+            outConfig.inPlace = 0;
+            outConfig.constant = false;
+            const SizeVector& outDims = layer->outData[0]->getTensorDesc().getDims();
+            outConfig.desc = TensorDesc(Precision::FP32, outDims, InferenceEngine::TensorDesc::getLayoutByDims(outDims));
+            config.outConfs.push_back(outConfig);
+
+            config.dynBatchSupport = false;
+            confs.push_back(config);
```

Comment:
It is not necessary to copy addConfig body to support custom precision. You can use overloaded DataConfigurator ctor. See https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/nodes/reverse_sequence.cpp#L62 as an example

Reply:
Done